bitnami/common/templates/_errors.tpl

```diff
@@ -14,7 +14,10 @@ Required password params:
 {{- define "common.errors.upgrade.passwords.empty" -}}
   {{- $validationErrors := join "" .validationErrors -}}
   {{- if and $validationErrors .context.Release.IsUpgrade -}}
-    {{- $errorString := "\nPASSWORDS ERROR: you must provide your current passwords when upgrade the release%s" -}}
+    {{- $errorString := "\nPASSWORDS ERROR: You must provide your current passwords when upgrading the release." -}}
+    {{- $errorString = print $errorString "\n                 Note that even after reinstallation old credentials may be needed as they may be kept in persistent volume claims." -}}
```

Comment:
I think the information of the sentence below is not relevant in the context of upgrading. That's information that a user can get from the documentation link. But if the user is doing an upgrade, it is not relevant to receive information about reinstallation.

```suggestion
```

Do you agree?

Reply:
This is from my personal experience. When the chart told me I needed to provide credentials, I decided reinstallation would be simpler since I didn't mind losing any data and getting a new password. I don't think I would have clicked the link in the proposed change without more context because I would have assumed there is a need for credentials only during upgrade. Of course the reality was I couldn't lose any data or have a new password this way anyway.

It is true that the information is provided in the link so I wouldn't mind deleting this, it's just something that I think would have helped me in my specific case and reasoning. I would understand if that is not the expected usual case so just say the word and I will accept the change.

Reply:
If it was useful for you, might be useful for others as well.